docs/courses/module.md

```diff
@@ -0,0 +1,90 @@
+# LibreLingo Documentation on Module
```

Comment:
```suggestion
# LibreLingo Course Format: Modules
```

and the same idea for all skill.md

what do you think?

I like to be more specific  ` LibreLingo Documentation` doesn't tell me if this is a user doc for the app, or about the course system etc.

Reply:
Yeah. This is one of the reasons I chose to use `LibreLingo Course Documentation` as the heading on README.md. But I forgot to continue the idea to other documents. Maybe:

- `LibreLingo Course Documentation` for the README.md
- `LibreLingo Course Documentation: Course` for course.md
- `LibreLingo Course Documentation: Module` for module.md
- `LibreLingo Course Documentation: Skill` for skill.md

It gives us an idea that where we are inside the course documentation hierarchy.

Reply:
Yeah, I originally didn't like (and still don't like) "LibreLingo Course Documentation" because I feel like "Course" is redundant (Most of LibreLingo is about courses) as well as doesn't clarify whether this is about creating courses or using them etc.

How about something like this?

- `Creating and editing LibreLingo courses` for the README.md
- `Creating and editing LibreLingo courses: Course` for course.md
- `Creating and editing LibreLingo courses: Module` for module.md
- `Creating and editing LibreLingo courses: Skill` for skill.md

Reply:
Well, we already have separate pages for creating and editing courses (`creating-courses.md` and `editing-courses.md`). So we'd have a problem assigning a title to them.

e.g. A title like: `Creating and editing LibreLingo courses: Editing Courses` in `editing-courses.md` might be long and confusing.

Also I've included reference docs for cource, module and skill. So the title `Creating and editing LibreLingo courses` wouldn't be totally on point with them.

Reply:
ok, let's leave it as you originally suggested and see how it works in real life!